app/libraries/ACL.php

```diff
@@ -2,9 +2,7 @@
 
 namespace core;
 namespace app\libraries;
-use core\Database;
 use \Exception as Exception;
```

Comment:
I'm not sure why you do this.   new Exception and new \Exception are the same thing. and you only use it once in this file.